vocab.py

```diff
@@ -130,3 +130,7 @@ def copy(self) -> Dict[str, Union[ValidDataSourceName, datetime]]:
             'data_source_name': self._data_source_name,
             'data_updated_at': self._data_updated_at
         }
+
+
+class PlacementType(IntEnum):
+    ZOOM = 0
```

Comment:
Wondering now if this should be `LTIPlacementType` or `LtiPlacementType` ...

Nit: This could also be moved up with the other enums to improve organization. Also, the `external_tool_id` might be a better ID for these types; I think that could be discussed if or when more LTI tools are reported on.

Reply:
Yeah, naming things is hard! I can put it first in the file though.

Reply:
Also maybe calling it placement isn't even accurate. It's really an internal id for this external tool. Without the table it kind of loses external context.

Reply:
I'm just going to leave this name for now, it might get changed in a future commit anyway when I think more about it.

Reply:
Yeah, you're right, `Placement` itself is confusing. I'm okay keeping it as is for now if you want to continue thinking about it.